### PR TITLE
Fix action button

### LINKED ISSUE #45: Add option in /admin/videos/ to add a video in Wagtail 2.10
If you go to /admin/videos/ after you add the first video, you don't have a way to add a new video in that interface.

it would be great to add a button `Add video` similar to the one find in /admin/images/

image

image

Wagtail: 2.10.1
wagtailvideos: 2.8.1

### PR DESCRIPTION
**Pull request by jkevingutierrez**
Fix #45

### CODE REVIEW

**Comment by seb-b** [comment]
Ah good pickup, I've left one comment.

So it looks like `wagtailadmin/shared/header.html` was changed in 2.10 to take a different parameter. (`add_link` -> `action_url`)

So this is the correct approach for 2.10+. We can either make a convoluted approach that loads a different template based on the wagtail version, or I'll make then next release require wt>2.10

File: wagtailvideos/templates/wagtailvideos/videos/index.html [hunk 1]
```diff
@@ -23,7 +23,8 @@
     {% trans "Videos" as video_str %}
 
     {% trans "Add a video" as add_video_str %}
-    {% include "wagtailadmin/shared/header.html" with title=video_str add_link="wagtailvideos:add_multiple" icon="media" add_text=add_video_str search_url="wagtailvideos:index" %}
+    {% url "wagtailvideos:add_multiple" as add_link %}
```
Comment (seb-b) [inline wagtailvideos/templates/wagtailvideos/videos/index.html:26]: This should be the name of the route, not a reversed url, it's causing the tests to fail: https://travis-ci.org/github/neon-jungle/wagtailvideos/builds/723598095

You can change the below line to:
```
{% include "wagtailadmin/shared/header.html" with
     title=video_str 
     action_url="wagtailvideos:add_multiple"
     icon="media"
     action_text=add_video_str 
     search_url="wagtailvideos:index" %}
```

**Comment by jkevingutierrez** [comment]
It could also use both parameters, like:

{% include "wagtailadmin/shared/header.html" with
     title=video_str 
     add_link="wagtailvideos:add_multiple"
     action_url="wagtailvideos:add_multiple"
     icon="media"
     add_text=add_video_str 
     action_text=add_video_str 
     search_url="wagtailvideos:index" %}

What do you think?

File: wagtailvideos/templates/wagtailvideos/videos/index.html (hunk 1 above)
Comment (jkevingutierrez) [reply wagtailvideos/templates/wagtailvideos/videos/index.html:26]: Got it, I was using https://github.com/wagtail/wagtail/blob/e097cfaab25bfd0b44a5133f4a43a330a0a16629/wagtail/images/templates/wagtailimages/images/index.html#L29 as base, so I thought that it needed to be the reversed url

Comment (seb-b) [reply wagtailvideos/templates/wagtailvideos/videos/index.html:26]: oh yeah you're right, not sure what I was thinking there. Looking at the [template code](https://github.com/wagtail/wagtail/blob/v2.10.1/wagtail/admin/templates/wagtailadmin/shared/header.html) it definitely should be a fully reversed url, it must be something else tripping up the tests

**Comment by seb-b** [comment]
@jkevingutierrez Yep, thats a good workaround, happy for both parameters to be used :)

File: wagtailvideos/templates/wagtailvideos/videos/index.html (hunk 1 above)
Comment (seb-b) [reply wagtailvideos/templates/wagtailvideos/videos/index.html:26]: Oh I see what it is now, in 2.10 it takes a reversed url but before that it was a just a string you can see 2.9 here: 
https://github.com/wagtail/wagtail/blob/v2.9.3/wagtail/admin/templates/wagtailadmin/shared/header.html

Comment (jkevingutierrez) [reply wagtailvideos/templates/wagtailvideos/videos/index.html:26]: Fixed :D

**Comment by jkevingutierrez** [comment]
@seb-b done

**Comment by seb-b** [comment]
Released in 2.8.2 :tada:

**Pull request merged**